docs/content/manual/_incoming/1418-2383-topology-scheduling-of-guest-cluster-workloads.md

```diff
@@ -12,17 +12,11 @@ title: Topology aware scheduling of guest cluster workloads
 1. Environment preparation as above steps
 1. Access Harvester node config page 
 1. Add the following node labels with values
- - topology.kubernetes.io/zone
- - topology.kubernetes.io/region
+- topology.kubernetes.io/zone
+- topology.kubernetes.io/region
 1. Provision an RKE2 cluster 
 1. Wait for the provisioning complete
 1. Access RKE2 guest cluster 
```

Comment:
The number was reset here.

Reply:
Thanks for the check, I indent the previous line 15, 16 to make the follow up numbering can proceed.